README.md

```diff
@@ -332,6 +332,27 @@ end
 # => ["OK", 1]
 ```
 
+#### Exception management
+
+The `raise_exception` flag in the `#pipelined` method of `RedisClient` is a feature that modifies the pipeline execution
+behavior. When set to `false`, it doesn't raise an exception when a command error occurs. Instead, it allows the
+pipeline to execute all commands, and any failed command will be available in the returned array. (Defaults to `true`)
+
+```ruby
+results = redis.pipelined(raise_exception: false) do |pipeline|
```

Comment:
```suggestion
results = redis.pipelined(exception: false) do |pipeline|
```

I think I'd prefer the shorter `exception: false`, which also map nicely with various Ruby core methods like `IO#read_nonblock(exception: false)`